app/src/main/res/layout/fragment_home.xml

```diff
@@ -5,5 +5,13 @@
     <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent">
+        <TextView
+            android:id="@+id/EmailText"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/spacing_medium"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent"/>
```

Comment:
This is a placeholder view, but don't forget to separate the first and last row in this block with a blank space

Reply:
You can use the Android Studio tool to make code reformat, you have go to Code -> Reformat Code.